decidim-core/app/views/decidim/participatory_process_steps/_step.html.erb

```diff
@@ -0,0 +1,21 @@
+<% # TODO: Move to decorator? %>
+<% status_class = step.end_date > Time.zone.now  ? "timeline__item--inactive" : "" %>
+<% status_class = step.active? ? "timeline__item--current" : status_class %>
+
+<li class="<%= "timeline__item #{status_class}" %>">
+  <div class="timeline__phase">
+    <span class="timeline__phase__number"><%= step.position %></span>
+  </div>
+  <div class="timeline__info">
+    <div class="timeline__header">
+      <span class="timeline__date text-small">
+        <%= step.start_date.strftime("%d/%m/%Y") %> - <%= step.end_date.strftime("%d/%m/%Y") %>
+      </span>
+      <h6 class="timeline__title heading6"><%= t('.info') %></h6>
+    </div>
+    <div class="timeline__content">
+      <% #TODO short_description? %>
```

Comment:
We'll be unifying `short_description` and `description` in the near future. Probably `description` then.